region/norfair/crocomire/Indiana Jones Room.json

```diff
@@ -711,6 +711,279 @@
         "Store a shinecharge and start running below the left side of the vertical door, to set up the jump in a way that shinesparking at the apex of the jump approximately aligns with the top of the far door."
       ]
     },
+    {
+      "link": [2, 1],
+      "name": "Leave Spinning (Short Runway, Space Jump)",
+      "requires": [
+        {"obstaclesCleared": ["B", "E"]},
+        "canPreciseSpaceJump"
+      ],
+      "exitCondition": {
+        "leaveSpinning": {
+          "remoteRunway": {
+            "length": 32,
+            "openEnd": 1,
+            "gentleUpTiles": 6
+          }
+        }
+      },
+      "unlocksDoors": [{"types": ["ammo"], "requires": [], "useImplicitRequires": false}],
+      "flashSuitChecked": true
+    },
+    {
+      "link": [2, 1],
+      "name": "Leave Spinning (Long Runway, Space Jump)",
+      "requires": [
+        {"obstaclesCleared": ["A", "B", "E"]},
+        "canPreciseSpaceJump"
+      ],
+      "exitCondition": {
+        "leaveSpinning": {
+          "remoteRunway": {
+            "length": 45,
+            "openEnd": 1
+          }
+        }
+      },
+      "unlocksDoors": [{"types": ["ammo"], "requires": [], "useImplicitRequires": false}],
```

Comment:
What's the problem with implicitRequires in this room?

Reply:
The obstacle "E" represents that the door has already been opened (by "Open Door" strat 78). So its unlock requirements would have already been processed there. The `useImplicitRequires` here makes it so that the ammo requirements to unlock the door don't get doubled up.